boons/boons.yml

```diff
@@ -416,7 +416,7 @@
     When you successfully invoke this boon, you gain information from an object or place within range as follows:
     <ul>
     <li><strong>Power Level 5</strong> - The target can read vague ideas and impressions to learn what took place near the object or place within the past hour.</li>
-    <li><strong>Power Level 6</strong> - The target can see a vivid vision, similar to a recording, of what took place near the object or place within the last hour.</li>
+    <li><strong>Power Level 6</strong> - The target can read vague ideas and impressions to learn what took place near the object or place within the past hour.</li>
```

Comment:
@oconnor0 I almost accepted this without review since you've done so well with proofreading, but I have no idea why this was changed? You've overwritten Power Level 6 with Power Level 5, I think